content/applications/hr/lunch/vendors.rst

```diff
@@ -0,0 +1,152 @@
+   :align: center
+   :alt: The top portion of the vendor form filled out.
+
+.. _lunch/availability:
+
+Availability
+============
+
+The :guilabel:`AVAILABILITY` section presents a table with two rows. The days of the week populate
+the top row, and the bottom row has checkboxes. Tick the corresponding checkbox for each day of the
+week the vendor is available.
+
+By default, Monday through Friday are ticked.
+
+.. image:: vendors/availability.png
+   :align: center
+   :alt: The default view of the availability section, with Mon-Fri enabled.
+
+Orders
+======
+
+The *Orders* section of the vendor form details which locations the vendor is available for, and how
+and when orders are placed and received.
+
+- :guilabel:`Delivery`: using the drop-down menu, select either :guilabel:`Delivery` if the vendor
+  delivers to the office, or :guilabel:`No Delivery` if orders must be picked up.
+- :guilabel:`Location`: select which locations are able to order from this vendor. Multiple
+  locations can be selected. If this field is left blank, **all** locations can order from the
+  vendor.
+
+  .. note::
+     By default, an `HQ Office` location is created by default when creating a database, and is
+     available to select from the list.
+
+- :guilabel:`Send Order By`: click the radio button to select how orders are sent to the vendor. The
+  available options are :guilabel:`Phone` or :guilabel:`Email`.
+- :guilabel:`Order Time`: this field **only** appears if :guilabel:`Email` is selected for
+  :guilabel:`Send Order By`. Enter the time that an order must be emailed for the order to be
+  accepted. Enter the time in a `HH:MM` format, then select either :guilabel:`AM` or :guilabel:`PM`
+  from the drop-down next to the time field.
+
+.. image:: vendors/orders.png
+   :align: center
+   :alt: The orders section of a vendor form, with all fields filled out.
+
+.. _lunch/extras:
+
+Extras
+======
+
+When ordering an item in the *Lunch* app, it is possible to show extra items that can be added to an
+order, sometimes referred to as *add-ons*. These can be configured in any manner that suits the
+products being offered.
+
+By default, Odoo allows for three types of extra items, which can be thought of as *categories*. By
+default, the first type or *category* of add-ons is labeled `Extras`, the second is labeled
+`Beverages`, and the third is labeled `Extra Label 3`.
+
+.. important::
+   When configuring the extras, it is important to keep in mind that all the extras configured will
+   appear for **every item** offered by the vendor. That means that only items which apply to
+   **all** products from the vendor should be added.
+
+.. _lunch/configure-extras:
+
+Configure extras
+----------------
+
+Enter the following information for each of the three available extra sections:
+
+- :guilabel:`Extra (#) Label`: enter a name for the type of extra, such as `Toppings` or
+  `Beverages`. This can be thought of as a *category*.
```

Comment:
```suggestion
- :guilabel:`Extra (#) Label`: enter a name for the type of extra, such as `Toppings`. This can be thought of as a *category*.
```
I removed this since there is already a "beverages" category, right?